docker/grafana/datasource.yaml

```diff
@@ -3,7 +3,11 @@ datasources:
     type: prometheus
     access: proxy
     url: http://prometheus01:9090
-  - name: M3Query
+  - name: M3Query - Prometheus
     type: prometheus
     access: proxy
     url: http://m3coordinator01:7201
+  - name: M3Query - Graphite
+    type:  graphite
+    access: proxy
+    url: http://m3coordinator01:7201/api/v1/graphite
```

Comment:
Grafana doesn't add anything like it does with Prom?

Reply:
What do you mean? The reason the default sources get loaded into grafana with `start_m3` is because my script loads this file into Grafanas config

Reply:
I meant for the URL - e.g. if you chose Prom, it injects the `/api/v1/query_range`, etc.

Reply:
oh, I think for this one it does not (I ran m3_stack and everything worked properly)